app/serializers/organization_serializer.rb

```diff
@@ -24,5 +24,5 @@
 #
 class OrganizationSerializer
   include JSONAPI::Serializer
-  attributes :name, :address, :email, :phone, :welcome_text, :about_us_text
+  attributes :name, :address, :email, :phone, :welcome_text, :about_us_text, :organization_slides
```

Comment:
Please check this link:
https://github.com/jsonapi-serializer/jsonapi-serializer#links-on-a-relationship